### ISSUE TITLE
Do i get  soft ban for walking same speed for a long period of time?>

### ISSUE THREAD

**Issue by NCAO93** [finndev/PokeBuddy#660]
or how often do i change my speed. give me an example. 
 i have checked other post didnt find it so i  start a new post

**Comment by curtiola** [comment]
I don't think keeping the same speed will be an issue, I have seen posts recommending keeping the speed below 12 though

**Comment by NCAO93** [comment]
yeah i read ur post, and others kindna confusing me, SO i had soft ban, from gps spooking  i reset it from the poke stop, but i had this specific location that is like 3-5h away from my area i want to bot AT. when i can do that spot?

**Comment by hashkikker** [comment]
look up the coordinates at itouchmap.com  so u can go there:)

**Comment by curtiola** [comment]
I don't know for sure, but you can download quickspin, set the coords for the spot you want to go and have it un-softban you, then close quickspin and open pokebuddy, set those coords and you should be fine

**Comment by NCAO93** [comment]
which website u download quickspin from @curtiola

**Comment by curtiola** [comment]
Google knows, I don't lol I never change locations so I've never needed it, I just know it works from the testimony of others, sorry
